diagonal3D:
  filter_dev:
    xfb_a: &id001
    - 1.0
    - -1.8327128937800907
    - 0.8327128937800906
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    xfb_b: &id002
    - -1562.5000000000002
    - 5975.799251222468
    - -8578.02704428525
    - 5465.8416895941755
    - -1301.1138965313917
    - 0.0
    - 0
    xff_taps: &id003
    - 4.3281348860582106e-05
    - 7.425462038955154e-05
    - 9.705279565984172e-05
    - 0.0001143268664277455
    - 0.00012779372648337087
    - 0.0001385589572482024
    - 0.00014735510316024963
    - 0.0001546655680398956
    - 0.00016081606581876403
    - 0.00016603060850264173
    - 0.0001704672017898226
    - 0.00017424068446945072
    - 0.00017743736159559086
    - 0.0001801243953444589
    - 0.00018235585120786286
    - 0.00018417661165903988
    - 0.0001856249294141912
    - 0.00018673411110192842
    - 0.00018753364348760463
    - 0.00018804996075949348
    - 0.00018830697957246932
    - 0.00018832648263869222
    - 0.00018812840271833242
    - 0.00018773104014821113
    - 0.00018715123539561496
    - 0.0001864045104370723
    - 0.0001855051879482078
    - 0.00018446649367627913
    - 0.00018330064670904825
    - 0.00018201894067953052
    - 0.0001806318151001121
    - 0.00017914892142681594
    - 0.00017757918300067408
    - 0.0001759308499884468
    - 0.0001742115500177024
    - 0.0001724283348658537
    - 0.00017058772360820047
    - 0.00016869574243108003
    - 0.00016675796148438556
    - 0.00016477952908931475
    - 0.000162765203902187
    - 0.00016071938563773217
    - 0.00015864614514308304
    - 0.00015654925433792433
    - 0.00015443221597827305
    - 0.00015229829186063642
    - 0.00015015052640608446
    - 0.00014799176106445472
    - 0.0001458246346466001
    - 0.00014365156482683587
    - 0.00014147474324239717
    - 0.00013929614455615005
    - 0.0001371175624232851
    - 0.0001349406141227609
    - 0.00013276677713195846
    - 0.00013059746790775785
    - 0.00012843404589925754
    - 0.00012627778455613574
    - 0.0001241298408838617
    - 0.00012199125511290192
    - 0.00011986297601080158
    - 0.00011774588598799904
    - 0.00011564081139765582
    - 0.00011354852305447012
    - 0.00011146973434334344
    - 0.00010940510120370378
    - 0.00010735522392412741
    - 0.00010532065052791985
    - 0.00010330188053278828
    - 0.00010129936897348744
    - 9.931352991068699e-05
    - 9.734473947537691e-05
    - 9.539333749158629e-05
    - 9.345963095404296e-05
    - 9.154390750459087e-05
    - 8.964642342379998e-05
    - 8.776740865473252e-05
    - 8.590706977048758e-05
    - 8.406559189567226e-05
    - 8.22431403738724e-05
    - 8.043986208479964e-05
    - 7.865588672739415e-05
    - 7.689132775341303e-05
    - 7.514628301723653e-05
    - 7.342083810437626e-05
    - 7.171506543010763e-05
    - 7.002902485144787e-05
    - 6.836276485046062e-05
    - 6.671632332034412e-05
    - 6.508972833615046e-05
    - 6.348299873890847e-05
    - 6.189614478469334e-05
    - 6.032916864893952e-05
    - 5.878206502663585e-05
    - 5.725482163667861e-05
    - 5.574741990536697e-05
    - 5.4259833764845715e-05
    - 5.2792031836243506e-05
    - 5.1343975988000466e-05
    - 4.9915621623199095e-05
    - 4.850692405741737e-05
    - 4.711783276628813e-05
    - 4.574829197875573e-05
    - 4.43982414850428e-05
    - 4.306761711801917e-05
    - 4.175635112710875e-05
    - 4.0464372361052505e-05
    - 3.919160653830505e-05
    - 3.7937976347268854e-05
    - 3.670340147662721e-05
    - 3.548779794217408e-05
    - 3.4291078157522684e-05
    - 3.311315414285827e-05
    - 3.1953936949881705e-05
    - 3.0813333907254525e-05
    - 2.9691249838469443e-05
    - 2.8587587455681704e-05
    - 2.7502247625455374e-05
    - 2.6435129423795432e-05
    - 2.5386130291784523e-05
    - 2.4355146050633515e-05
    - 2.334207102617228e-05
    - 2.2346798130889293e-05
    - 2.1369218800883178e-05
    - 2.0409223053837984e-05
    - 1.9466699675452504e-05
    - 1.85415362428576e-05
    - 1.7633619268170853e-05
    - 1.6742834190375617e-05
    - 1.5869065492972495e-05
    - 1.501219666192454e-05
    - 1.4172110261240014e-05
    - 1.3348687829967574e-05
    - 1.2541809884523099e-05
    - 1.1751355737914625e-05
    - 1.0977203477493576e-05
    - 1.0219229955156074e-05
    - 9.477311529922702e-06
    - 8.751319881222916e-06
    - 8.041117457835446e-06
    - 7.346561713369155e-06
    - 6.667522617043139e-06
    - 6.0038655889298155e-06
    - 5.355451998390861e-06
    - 4.722138455956871e-06
    - 4.103775311978411e-06
    - 3.5002034231458737e-06
    - 2.9112492229372998e-06
    - 2.3367193916319397e-06
    - 1.77640849708763e-06
    - 1.2301749681578922e-06
    - 6.97913524272982e-07
    - 1.7960368170736484e-07
    - -3.248226929747955e-07
    - -8.157944254625405e-07
    - -1.2937993789314704e-06
    - -1.7589556930319002e-06
    - -2.2112623904757632e-06
    - -2.6507970604257984e-06
    - -3.0777456780233917e-06
    - -3.4923007599470327e-06
    - -3.894638359659866e-06
    - -4.2849197606224235e-06
    - -4.663296882559255e-06
    - -5.0299167325965365e-06
    - -5.384923888297206e-06
    - -5.728461915808886e-06
    - -6.06067384142936e-06
    - -6.381702471180083e-06
    - -6.691690333232177e-06
    - -6.990779775291136e-06
    - -7.279112812364429e-06
    - -7.556831156378636e-06
    - -7.82407595884731e-06
    - -8.080987669446397e-06
    - -8.32770557886913e-06
    - -8.564368576649322e-06
    - -8.79112146560686e-06
    - -9.008106522991817e-06
    - -9.215465271588696e-06
    - -9.41333879423503e-06
    - -9.601866599944433e-06
    - -9.781187808461574e-06
    - -9.951441272185948e-06
    - -1.0112765247929183e-05
    - -1.026529763901506e-05
    - -1.0409176165394353e-05
    - -1.0544538198840157e-05
    - -1.06715208563355e-05
    - -1.079026084624207e-05
    - -1.0900894571885819e-05
    - -1.1003557981648248e-05
    - -1.1098386678007182e-05
    - -1.1185515768854972e-05
    - -1.1265079980789273e-05
    - -1.1337213511916817e-05
    - -1.1402050150803852e-05
    - -1.1459723133423086e-05
    - -1.1510365271210444e-05
    - -1.1554108815794926e-05
    - -1.1591085597867883e-05
    - -1.1621426895295817e-05
    - -1.1645263563349605e-05
    - -1.1662725855747926e-05
    - -1.1673943438472922e-05
    - -1.1679044965536875e-05
    - -1.1678157672397229e-05
    - -1.167140639463355e-05
    - -1.1658912920006154e-05
    - -1.1640796998408463e-05
    - -1.16171925052761e-05
    - -1.1588230809297217e-05
    - -1.1554038582918859e-05
    - -1.1514739297618784e-05
    - -1.1470454981560157e-05
    - -1.1421306912614505e-05
    - -1.1367416016106168e-05
    - -1.1308902689247953e-05
    - -1.1245886824205562e-05
    - -1.1178487534794501e-05
    - -1.1106823210998628e-05
    - -1.1031011316004333e-05
    - -1.0951168512281153e-05
    - -1.0867410510519428e-05
    - -1.0779852228317562e-05
    - -1.0688607651598621e-05
    - -1.0593789995020994e-05
    - -1.0495511557334594e-05
    - -1.039388387736e-05
    - -1.0289017587469629e-05
    - -1.0181022574064708e-05
    - -1.0070007832051099e-05
    - -9.956081604713151e-06
    - -9.83935114904579e-06
    - -9.719922659049695e-06
    - -9.597900534514229e-06
    - -9.473387133321401e-06
    - -9.346483778527367e-06
    - -9.217292161716977e-06
    - -9.085913297972333e-06
    - -8.952446803259335e-06
    - -8.816991004923873e-06
    - -8.67964300657758e-06
    - -8.540498505667733e-06
    - -8.399651912888213e-06
    - -8.25719610807197e-06
    - -8.113222426822596e-06
    - -7.967822960741373e-06
    - -7.821089185243012e-06
    - -7.67311101694934e-06
    - -7.5239770983659886e-06
    - -7.373774708591318e-06
    - -7.2225899599449845e-06
    - -7.07050766201114e-06
    - -6.917611491855646e-06
    - -6.763983838490781e-06
    - -6.609705961136611e-06
    - -6.454857828642078e-06
    - -6.299518271869471e-06
    - -6.143764812683087e-06
    - -5.987673812091679e-06
    - -5.831320306102929e-06
    - -5.674778164570759e-06
    - -5.518119931038671e-06
    - -5.3614169752558475e-06
    - -5.204739326320213e-06
    - -5.048155823542518e-06
    - -4.8917339501402185e-06
    - -4.735539985375715e-06
    - -4.579638839670278e-06
    - -4.424094208145475e-06
    - -4.268968407191723e-06
    - -4.114322529387356e-06
    - -3.960216281449708e-06
    - -3.806708140378107e-06
    - -3.6538551924901013e-06
    - -3.5017132902060353e-06
    - -3.350336891343822e-06
    - -3.199779215249425e-06
    - -3.050092080576154e-06
    - -2.9013260584843495e-06
    - -2.7535303065072796e-06
    - -2.6067527210282935e-06
    - -2.4610397725866365e-06
    - -2.3164366612354496e-06
    - -2.1729872259865993e-06
    - -2.03073420543537e-06
    - -1.8897193577193018e-06
    - -1.7499845222871458e-06
    - -1.6115763741883357e-06
    - -1.4745402013048758e-06
    - -1.3389168118352116e-06
    - -1.2047429172539572e-06
    - -1.0720513531819161e-06
    - -9.408717676489959e-07
    - -8.112310884339104e-07
    - -6.831543158336095e-07
    - -5.566649281379802e-07
    - -4.3178547329566936e-07
    - -3.0853769283767617e-07
    - -1.8694281550996008e-07
    - -6.702141943136437e-08
    - 5.120646791614669e-08
    - 1.677216207296416e-07
    - 2.825055513520847e-07
    - 3.9554073389277333e-07
    - 5.06810468257742e-07
    - 6.16299005716511e-07
    - 7.239913086610309e-07
    - 8.298730830238706e-07
    - 9.339304682660414e-07
    - 1.0361500341132989e-06
    - 1.1365184756550585e-06
    - 1.2350226532525356e-06
    - 1.3316493602714834e-06
    - 1.4263854516340686e-06
    - 1.519217700215031e-06
    - 1.6101329977334376e-06
    - 1.69911825317422e-06
    - 1.786160595736407e-06
    - 1.8712472331497422e-06
    - 1.9543655821090895e-06
    - 2.035504683521064e-06
    - 2.1146578480494704e-06
    - 2.1918193324026767e-06
    - 2.266984308421006e-06
    - 2.340148506708278e-06
    - 2.411308379858224e-06
    - 2.480460971525749e-06
    - 2.54760409246017e-06
    - 2.6127361863106957e-06
    - 2.6758565074339476e-06
    - 2.736964936000518e-06
    - 2.7960621375281814e-06
    - 2.8531494044323984e-06
    - 2.908228809269507e-06
    - 2.9613030424791095e-06
    - 3.012375563970399e-06
    - 3.061450444177022e-06
    - 3.1085325090046604e-06
    - 3.153627176783391e-06
    - 3.196740609162852e-06
    - 3.237879547813387e-06
    - 3.2770514652117853e-06
    - 3.3142644012562365e-06
    - 3.3495271138796863e-06
    - 3.382848915222108e-06
    - 3.4142398214031337e-06
    - 3.4437103880885354e-06
    - 3.471271860419759e-06
    - 3.4969360085087734e-06
    - 3.5207152768527254e-06
    - 3.5426226190418356e-06
    - 3.5626716463198596e-06
    - 3.580876461333013e-06
    - 3.5972518057450475e-06
    - 3.6118128929600183e-06
    - 3.6245755547555513e-06
    - 3.6355560729668075e-06
    - 3.6447713251700112e-06
    - 3.6522386153707988e-06
    - 3.6579758188558185e-06
    - 3.662001212050201e-06
    - 3.6643336167157888e-06
    - 3.664992229193743e-06
    - 3.6639967642069053e-06
    - 3.6613672837037317e-06
    - 3.6571243404575823e-06
    - 3.651288806678252e-06
    - 3.6438820175442088e-06
    - 3.6349255996454174e-06
    - 3.624441614448597e-06
    - 3.6124523864939905e-06
    - 3.5989806466966097e-06
    - 3.5840493603562576e-06
    - 3.567681870901795e-06
    - 3.549901729737467e-06
    - 3.5307328453255922e-06
    - 3.510199324591826e-06
    - 3.4883256431020375e-06
    - 3.465136515371781e-06
    - 3.440657082201846e-06
    - 3.4149127198892776e-06
    - 3.3879289545731177e-06
    - 3.35973077859044e-06
    - 3.3303438789345753e-06
    - 3.2997942182476916e-06
    - 3.268108056480601e-06
    - 3.2353117399246907e-06
    - 3.2014318417776382e-06
    - 3.1664949979042895e-06
    - 3.130528061018293e-06
    - 3.0935579374598822e-06
    - 3.05561173878712e-06
    - 3.0167166154364978e-06
    - 2.976899905704224e-06
    - 2.936188967262535e-06
    - 2.894611324467238e-06
    - 2.85219449843844e-06
    - 2.8089661546677126e-06
    - 2.7649539311566677e-06
    - 2.7201855817960034e-06
    - 2.674688803299396e-06
    - 2.6284913784425063e-06
    - 2.581621002435758e-06
    - 2.5341054269639176e-06
    - 2.4859722899268833e-06
    - 2.4372492676567214e-06
    - 2.38796392527374e-06
    - 2.338143898737042e-06
    - 2.287816771012725e-06
    - 2.2370103011335584e-06
    - 2.1857523584528027e-06
    - 2.134071158697604e-06
    - 2.0819950563440953e-06
    - 2.029552272143633e-06
    - 1.976769465031058e-06
    - 1.9236690445857574e-06
    - 1.8702644536783252e-06
    - 1.8165830199249644e-06
    - 1.7626576768738958e-06
    - 1.7085200966174922e-06
    - 1.654199147346706e-06
    - 1.599721843212625e-06
    - 1.5451140555877823e-06
    - 1.4904012173246066e-06
    - 1.435608406044488e-06
    - 1.3807605497398527e-06
    - 1.3258822215098328e-06
    - 1.2709977241706064e-06
    - 1.2161308678364998e-06
    - 1.161305088272355e-06
    - 1.106543273822608e-06
    - 1.0518679288680632e-06
    - 9.973010329549527e-07
    - 9.428642221978815e-07
    - 8.885786553908013e-07
    - 8.344651977869646e-07
    - 7.805442936227284e-07
    - 7.268361703325525e-07
    - 6.733607538449312e-07
    - 6.20137933834922e-07
    - 5.671875148447839e-07
    - 5.145293596706451e-07
    - 4.6218282162214013e-07
    - 4.101659871321632e-07
    - 3.5849570563568036e-07
    - 3.0718918540693093e-07
    - 2.5626388494359486e-07
    - 2.0573733251341428e-07
    - 1.556267005547462e-07
    - 1.0594883695040403e-07
    - 5.672007450841897e-08
    - 7.956401078091935e-09
    - -4.0326673188252225e-08
    - -8.811393279652987e-08
    - -1.3539059532958762e-07
    - -1.8214213900952774e-07
    - -2.2835445081357798e-07
    - -2.740136613387212e-07
    - -3.191062977794083e-07
    - -3.636191219047937e-07
    - -4.075392848098563e-07
    - -4.5085416581613516e-07
    - -4.935515273080274e-07
    - -5.356193531221292e-07
    - -5.770460026380721e-07
    - -6.178200502479737e-07
    - -6.57930447345061e-07
    - -6.973663712708727e-07
    - -7.361173886125163e-07
    - -7.74173296087329e-07
    - -8.11524274586122e-07
    - -8.481607301861788e-07
    - -8.84073453267549e-07
    - -9.192534631157515e-07
    - -9.536921689025564e-07
    - -9.873812152399134e-07
    - -1.0203126436396647e-06
    - -1.0524787383572533e-06
    - -1.0838721880007086e-06
    - -1.1144859314585527e-06
    - -1.1443133194872226e-06
    - -1.17334796056125e-06
    - -1.2015838822758586e-06
    - -1.229015376945861e-06
    - -1.2556371626144794e-06
    - -1.2814442281482797e-06
    - -1.3064319935347586e-06
    - -1.330596154139493e-06
    - -1.3539328401713346e-06
    - -1.3764384608645164e-06
    - -1.3981098651632822e-06
    - -1.4189441858125402e-06
    - -1.438938997953654e-06
    - -1.4580921616058982e-06
    - -1.476401979089399e-06
    - -1.4938670366581838e-06
    - -1.5104863611507805e-06
    - -1.5262592609560284e-06
    - -1.5411854820113547e-06
    - -1.5552650483577785e-06
    - -1.5684984182101013e-06
    - -1.5808863245794274e-06
    - -1.5924299308218159e-06
    - -1.6031306708086029e-06
    - -1.612990404200873e-06
    - -1.6220112565498566e-06
    - -1.6301957745578815e-06
    - -1.6375467662567569e-06
    - -1.6440674563364497e-06
    - -1.649761326411016e-06
    - -1.654632270327462e-06
    - -1.6586844342327915e-06
    - -1.6619223712520947e-06
    - -1.6643508810550812e-06
    - -1.6659751643866925e-06
    - -1.6668006631082238e-06
    - -1.6668332165571097e-06
    - -1.6660789023115058e-06
    - -1.6645441913391397e-06
    - -1.6622357877141772e-06
    - -1.659160783027796e-06
    - -1.6553264955354037e-06
    - -1.6507406240013552e-06
    - -1.64541108619334e-06
    - -1.639346171908885e-06
    - -1.6325543804670982e-06
    - -1.6250445726239573e-06
    - -1.6168258072882218e-06
    - -1.6079074938951407e-06
    - -1.5982992298635785e-06
    - -1.5880109521497434e-06
    - -1.5770527731220469e-06
    - -1.5654351309085075e-06
    - -1.5531686244164642e-06
    - -1.5402641630176175e-06
    - -1.5267328010263343e-06
    - -1.5125858869104482e-06
    - -1.4978348973657693e-06
    - -1.4824915861826944e-06
    - -1.4665678180481955e-06
    - -1.4500757172088675e-06
    - -1.4330275011528583e-06
    - -1.4154356292225117e-06
    - -1.3973126363336645e-06
    - -1.3786712816997927e-06
    - -1.359524382745988e-06
    - -1.3398849641011274e-06
    - -1.3197660918637118e-06
    - -1.299181023017541e-06
    - -1.2781430402145873e-06
    - -1.2566656018027435e-06
    - -1.2347621773297268e-06
    - -1.2124463983556185e-06
    - -1.1897318946339644e-06
    - -1.1666324449899845e-06
    - -1.1431618134376357e-06
    - -1.1193339007294616e-06
    - -1.0951625814393513e-06
    - -1.0706618562614391e-06
    - -1.0458456896515594e-06
    - -1.0207281625530167e-06
    - -9.9532331037352e-07
    - -9.696452759633228e-07
    - -9.437081477742951e-07
    - -9.17526112937481e-07
    - -8.911132954545994e-07
    - -8.644839092202356e-07
    - -8.376520960976835e-07
    - -8.106320787548691e-07
    - -7.834379985143909e-07
    - -7.56084067918902e-07
    - -7.285844083181183e-07
    - -7.00953202161992e-07
    - -6.732045303351066e-07
    - -6.453525241904493e-07
    - -6.17411202647231e-07
    - -5.89394623941094e-07
    - -5.613167224014851e-07
    - -5.331914597150001e-07
    - -5.050326609434912e-07
    - -4.768541645007865e-07
    - -4.4866965611986493e-07
    - -4.204928157419945e-07
    - -3.923371475959969e-07
    - -3.6421612376360405e-07
    - -3.361430157420938e-07
    - -3.08131053468729e-07
    - -2.8019330120182993e-07
    - -2.523428997890635e-07
    - -2.2459302132696258e-07
    - -1.96956948653136e-07
    - -1.6944703692448283e-07
    - -1.4207571483281386e-07
    - -1.148553327438384e-07
    - -8.779825425272849e-08
    - -6.091665247970044e-08
    - -3.422264331699382e-08
    - -7.728117147684905e-09
    - 1.8555108247665887e-08
    - 4.461544700094795e-08
    - 7.044138894242954e-08
    - 9.60216581516555e-08
    - 1.21345056972439e-07
    - 1.4640062506362428e-07
    - 1.7117748397248152e-07
    - 1.9566499676748547e-07
    - 2.1985261331355863e-07
    - 2.437300308684386e-07
    - 2.672870407504232e-07
    - 2.9051369098664353e-07
    - 3.134001358758107e-07
    - 3.359368025152986e-07
    - 3.58114245184003e-07
    - 3.79923317049555e-07
    - 4.0135502852616247e-07
    - 4.224007182684695e-07
    - 4.4305189935174985e-07
    - 4.6330039472393733e-07
    - 4.831381059817099e-07
    - 5.025570059369617e-07
    - 5.215486829491798e-07
    - 5.40104062171865e-07
    - 5.582128346571097e-07
    - 5.758638671270213e-07
    - 5.930479042611793e-07
    - 6.097702750884798e-07
    - 6.260292770137238e-07
    - 6.418181984821419e-07
    - 6.571290027822372e-07
    - 6.719544225227521e-07
    - 6.862882450861872e-07
    - 7.001252675328985e-07
    - 7.134608168450975e-07
    - 7.262906579765418e-07
    - 7.386106985255239e-07
    - 7.50417096736351e-07
    - 7.617061096766499e-07
    - 7.724742807052459e-07
    - 7.827183178190618e-07
    - 7.924352810788771e-07
    - 8.016224465752213e-07
    - 8.102774766877983e-07
    - 8.183982710771295e-07
    - 8.259831315409888e-07
    - 8.3303061466722e-07
    - 8.39539700252533e-07
    - 8.455096369883085e-07
    - 8.509400707765066e-07
    - 8.558307853709493e-07
    - 8.601816436285038e-07
    - 8.639921477866747e-07
    - 8.672616119825019e-07
    - 8.699900798082861e-07
    - 8.721791823708212e-07
    - 8.738308944839771e-07
    - 8.749472194087918e-07
    - 8.755300309269544e-07
    - 8.755813728683105e-07
    - 8.751034309116909e-07
    - 8.740987679964313e-07
    - 8.725702016859486e-07
    - 8.705209694896323e-07
    - 8.679545662269959e-07
    - 8.64874890973732e-07
    - 8.612860783610147e-07
    - 8.571926454075006e-07
    - 8.525993248833601e-07
    - 8.475112146933211e-07
    - 8.419336134734497e-07
    - 8.358721717328989e-07
    - 8.293327285520658e-07
    - 8.22321463062799e-07
    - 8.148447301806949e-07
    - 8.069092093725798e-07
    - 7.985217364823378e-07
    - 7.89689455535549e-07
    - 7.804196611480825e-07
    - 7.707199593183302e-07
    - 7.605981091242865e-07
    - 7.500621746984672e-07
    - 7.391203620108854e-07
    - 7.277811719717469e-07
    - 7.160532395144645e-07
    - 7.039454886912805e-07
    - 6.914669731309156e-07
    - 6.786270320746318e-07
    - 6.654351314047853e-07
    - 6.519010200301663e-07
    - 6.38034570978843e-07
    - 6.238459376930147e-07
    - 6.093453948288978e-07
    - 5.945434941303209e-07
    - 5.794509046418401e-07
    - 5.640785679196816e-07
    - 5.484375374102833e-07
    - 5.325391326685544e-07
    - 5.163947774048401e-07
    - 5.000161520524738e-07
    - 4.834150304700176e-07
    - 4.6660343334840103e-07
    - 4.4959346584153905e-07
    - 4.3239747010490044e-07
    - 4.15027861744035e-07
    - 3.974972813735513e-07
    - 3.7981843027366215e-07
    - 3.6200422137779533e-07
    - 3.4406761439028976e-07
    - 3.260217663000722e-07
    - 3.0787986602951647e-07
    - 2.8965528517397555e-07
    - 2.713614132023482e-07
    - 2.53011808390847e-07
    - 2.3462003303399551e-07
    - 2.1619980450641481e-07
    - 1.9776483067814383e-07
    - 1.7932896132745158e-07
    - 1.6090602395041552e-07
    - 1.4250997566610934e-07
    - 1.2415473956253234e-07
    - 1.058543571687712e-07
    - 8.76228252288818e-08
    - 6.94742480335876e-08
    - 5.1422673674217826e-08
    - 3.348224575211455e-08
    - 1.5667039165427986e-08
    - -2.008786677477779e-09
    - -1.9531150301266044e-08
    - -3.688589491094263e-08
    - -5.405895131061011e-08
    - -7.103618513723179e-08
    - -8.780356020844535e-08
    - -1.043469861523533e-07
    - -1.2065248214218649e-07
    - -1.3670602505646874e-07
    - -1.5249371399786827e-07
    - -1.6800161963275878e-07
    - -1.8321595011538105e-07
    - -1.9812290061982556e-07
    - -2.1270881811290095e-07
    - -2.2696005049096334e-07
    - -2.408631119319232e-07
    - -2.5440453274258487e-07
    - -2.6757102527481155e-07
    - -2.803493341880523e-07
    - -2.927264026447785e-07
    - -3.0468922278339086e-07
    - -3.16225002025426e-07
    - -3.27321013625313e-07
    - -3.379647629745058e-07
    - -3.4814383812802897e-07
    - -3.5784607601784164e-07
    - -3.6705941287390495e-07
    - -3.7577205027284663e-07
    - -3.8397230540129055e-07
    - -3.9164877688697934e-07
    - -3.987901948827981e-07
    - -4.0538558664582496e-07
    - -4.1142412643473654e-07
    - -4.168953008236283e-07
    - -4.217887583511957e-07
    - -4.2609447443232366e-07
    - -4.298026004950797e-07
    - -4.3290362817751306e-07
    - -4.353882383858502e-07
    - -4.3724746595041765e-07
    - -4.3847254878978915e-07
    - -4.390550920726093e-07
    - -4.3898691678057477e-07
    - -4.3826022305865457e-07
    - -4.3686743777541656e-07
    - -4.348013763213879e-07
    - -4.3205508796964704e-07
    - -4.2862201422947174e-07
    - -4.2449582908547646e-07
    - -4.1967058949115017e-07
    - -4.141405638298357e-07
    - -4.0790036454532637e-07
    - -4.0094474982817885e-07
    - -3.9326871591979813e-07
    - -3.8486723834704625e-07
    - -3.7573527328337795e-07
    - -3.658673620598916e-07
    - -3.552574261234923e-07
    - -3.438980587976139e-07
    - -3.3177984289779376e-07
    - -3.1888990764431313e-07
    - -3.0521010153402247e-07
    - -2.907137408181586e-07
    - -2.7536087769670985e-07
    - -2.590902975951296e-07
    - -2.4180686458360837e-07
    - -2.2336005447799498e-07
    - -2.03508017309981e-07
    - -1.8185528637903016e-07
    - -1.5774474964467154e-07
    - -1.3006899881265439e-07
    - -9.695030274245657e-08
    - -5.5246393135455676e-08
    yfb_a: *id001
    yfb_b: *id002
    yff_taps: *id003
    zfb_a: *id001
    zfb_b: *id002
    zff_taps: *id003
